javascript/webpacker_react-npm-module/src/ujs.js

```diff
@@ -40,8 +40,7 @@ const ujs = {
   },
 
   turbolinks5(onMount, onUnmount) {
-    this.handleEvent('turbolinks:load', onMount, { once: true })
-    this.handleEvent('turbolinks:render', onMount)
+    this.handleEvent('turbolinks:load', onMount)
```

Comment:
Please, before making a change read through [this](https://sevos.io/2017/02/27/turbolinks-lifecycle-explained.html) 

This approach is just wrong if you use turbolinks caching feature